_app/gatsby-config.js

```diff
@@ -0,0 +1,58 @@
+const { dirname } = require('path')
+
+module.exports = {
+  siteMetadata: {
+    title: 'Tamburetei',
+    description: 'Fazendo de tamburete as cadeiras de CC@UFCG',
+    author: '@OpenDevUFCG',
+  },
+  plugins: [
+    'gatsby-plugin-react-helmet',
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        path: `${__dirname}/..`,
+        name: 'subjects',
+        ignore: [
+          '**/\\.*',
+          '**/_app',
+          '**/scripts',
+          filePath => {
+            if (
+              // ignore markdowns in the project root folder
+              /tamburetei$/i.test(dirname(filePath)) &&
+              /\.md$/.test(filePath)
+            ) {
+              return true
+            }
+
+            return false
+          },
+        ],
+      },
+    },
+    {
+      resolve: 'gatsby-source-filesystem',
+      options: {
+        path: `${__dirname}/content/assets`,
+        name: 'assets',
+      },
+    },
+    'gatsby-transformer-remark',
+    'gatsby-transformer-sharp',
+    'gatsby-plugin-sharp',
+    {
+      resolve: 'gatsby-plugin-manifest',
+      options: {
+        name: 'Tamburetei',
+        short_name: 'Tamburetei',
+        start_url: '/',
+        background_color: '#663399',
+        theme_color: '#663399',
+        display: 'standalone',
+        icon: 'content/assets/icon.svg',
+      },
+    },
+    'gatsby-plugin-offline',
```

Comment:
Esse plugin por algum motivo tá dando erro aqui `Cannot read property 'component---src-pages-index-jsx' of undefined` quando rodo `yarn develop`

Removi ele e sumiu o erro, to vendo aqui oq seria